lib/oxidized/config.rb

```diff
@@ -30,6 +30,7 @@ def self.load(cmd_opts={})
         asetus.default.rest          = '127.0.0.1:8888' # or false to disable
       asetus.default.vars          = {}             # could be 'enable'=>'enablePW'
       asetus.default.groups        = {}             # group level configuration
+      asetus.default.pid           = '/var/run/oxidized.pid' # location of PID file
```

Comment:
I think default path should be `File.join(Oxidized::Config::Root, 'pid')`. This lets us run `oxidized` with as minimum permissions as possible. It should not be necessary for the user that runs `oxidized` to have write permission to `/var/run/oxidized.pid`